src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add: function(num) {
+      this.result += num;
+
+      return this;
+    },
+    subtract: function(num) {
+      this.result -= num;
+
+      return this;
+    },
+    multiply: function(num) {
+      this.result *= num;
+
+      return this;
+    },
+    divide: function(num) {
+      if (num !== 0) {
+        this.result /= num;
+      } else {
+        return 'Error: Division by zero is not allowed';
```

Comment:
it's better to return an error object for making errors 
```suggestion
        return trow new Error( 'Error: Division by zero is not allowed');
```